cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/cluster/SqlClusteredAgentScheduler.kt

```diff
@@ -175,6 +175,9 @@ class SqlClusteredAgentScheduler(
       .filterNot { disabledAgents.contains(it.key) }
       .toMutableMap()
 
+    log.debug("Agents running: {}, agents disabled: {}. Picking next agents to run from: {}",
```

Comment:
I like the additional log. Do you want to keep the fix/shuffle separate?

Reply:
I'll make another PR for the shuffle